airbyte-webapp/src/locales/en.json

```diff
@@ -338,6 +339,7 @@
   "connection.transfer": "Transfer",
   "connection.linkCopied": "Link copied!",
   "connection.copyLogLink": "Copy link to log",
+  "connection.connectionDeletedView": "This connection has been deleted. No changes or syncs can be completed.",
```

Comment:
cc @andyjih 

After reading this in the UI it felt a little repetitive because both sentences are structured the same way with a passive(?) tense:

This connection has been deleted
No changes or syncs can be completed

I would suggest a change in the second sentence to something along the lines of: "You can’t make any changes or run syncs."

Reply:
@edmundito That seems fine to me.